openfe/tests/protocols/openmm_md/test_plain_md_protocol.py

```diff
@@ -15,13 +15,14 @@
     PlainMDProtocol, PlainMDProtocolUnit, PlainMDProtocolResult,
 )
 from openfe.protocols.openmm_utils.charge_generation import (
-    HAS_NAGL, HAS_OPENEYE, HAS_ESPALOMA
+    HAS_NAGL, HAS_OPENEYE, HAS_ESPALOMA_CHARGE
 )
 import json
 import openfe
 from openfe.protocols import openmm_md
 import pathlib
 import logging
+from openfe.tests.conftest import HAS_INTERNET, HAS_ESPALOMA
```

Comment:
```suggestion
from openfe.tests.conftest import HAS_ESPALOMA
```
Not using internet check anywhere?